packages/wallets/src/features/cashier/modules/Transfer/components/TransferFormAmountInput/TransferFormAmountInput.tsx

```diff
@@ -103,6 +104,13 @@ const TransferFormAmountInput: React.FC<TProps> = ({ fieldName }) => {
         }
     }, [amountConverterHandler, debouncedAmountValue, isSameCurrency]);
 
+    const onBlurHandler = useCallback(() => {
+        if (!isSameCurrency) {
+            amountConverterHandler(amountValue);
+        }
```

Comment:
Need to call this function before `setFieldValue('activeAmountFieldName', undefined)` for it to actually execute without exiting based on [this](https://github.com/binary-com/deriv-app/pull/13235/files#diff-6d6b3448dcf1fe1ee33d0ae101a7ca8e0f1008896b5585348219b3b50692fdedR70) check